src/runtime/pkg/katautils/config.go

```diff
@@ -1315,6 +1315,7 @@ func newRemoteHypervisorConfig(h hypervisor) (vc.HypervisorConfig, error) {
 		RemoteHypervisorSocket:  h.getRemoteHypervisorSocket(),
 		RemoteHypervisorTimeout: h.getRemoteHypervisorTimeout(),
 		DisableGuestSeLinux:     true, // The remote hypervisor has a different guest, so Guest SELinux config doesn't work
+		SharedFS:                config.NoSharedFS,
```

Comment:
I think this makes sense for now and if we need to make it configurable in future we can add it to the config file. This also highlights that we need some tests in peer pods, in addition to the kata containers one mentioned in https://github.com/kata-containers/kata-containers/issues/9652 for the other shared_fs=none setup configurations, for doing changes to configmap/secrets and checking they have been reflected correctly